docs/products/clickhouse/howto/enable-tiered-storage.rst

```diff
@@ -0,0 +1,77 @@
+Enable tiered storage in Aiven for ClickHouse®
+==============================================
+
+.. important::
+
+   Aiven for ClickHouse® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Learn how to enable the tiered storage feature on your project and activate it for specific tables.
+To check what the tiered storage is, how it works, and why use it, see :doc:`Tiered storage in Aiven for ClickHouse® </docs/products/clickhouse/concepts/clickhouse-tiered-storage>`.
+
+About enabling tiered storage
+-----------------------------
+
+To use the tiered storage feature, you need to enable it on the project level by contacting the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_ and set it up on the table level using SQL (via CLI, for example).
+
+Limitations
+'''''''''''
+
+* When :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>`, the tiered storage feature cannot be deactivated.
+
+  .. tip::
+
+    As a workaround, you can create a new table (without enabling the tiered storage) and copy the data from the original table (with the tiered storage :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>`) to the new table. As soon as the data is copied to the new table, you can remove the original table.
+
+* With the tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>`, it's not possible to connect to an external existing object storage or cloud storage bucket.
+
+Tools
+'''''
+
+To enable the tiered storage, use SQL and an SQL client (for example, the ClickHouse CLI client).
+
+Prerequisites
+-------------
+
+* You have an Aiven organization and at least one project.
+* You have a command line tool (:doc:`ClickHouse client </docs/products/clickhouse/howto/connect-with-clickhouse-cli>`) installed.
+* All maintenance updates are applied on your service (check on the **Overview** page of your service in Aiven Console).
+
+Enable tiered storage on a project
+----------------------------------
+
+To enable the tiered storage feature on your project, request it from the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+.. topic:: Result
+   
+   Your project supports the tiered storage feature, and you can enable the feature on tables for your Aiven for ClickHouse services.
```

Comment:
"Your project now supports tiered storage, and you can enable it for each table of your Aiven for ClickHouse services."